packages/api/emails/survey/html.pug

```diff
@@ -96,12 +100,12 @@ html
 
             hr
 
-            h3 DEADLINE:
+            strong(class="deadline") DEADLINE:
```

Comment:
Based on the ticket, it looks like this should be h3 sized still. The strong will shrink it to normal text size (though bold). Is that intentional?